### PR TITLE
[#1285] Made sure frameAnimation is not turned on when we export. Added ...

### PR DESCRIPTION
**Pull request by dseif**
...a todo flag to update this to be optional later

### CODE REVIEW

File: src/core/media.js [hunk 1]
```diff
@@ -261,7 +261,10 @@
           collectedEvents = collectedEvents.concat( _tracks[ i ].trackEvents );
         }
 
-        return _popcornWrapper.generatePopcornString( _popcornOptions, _url, _target, null, callbacks, scripts, collectedEvents );
+        /* TODO: determine if we need to turn on frameAnimation or not before calling generatePopcornString
+         * for now we default to off when exporting by passing an empty options object
+         */
+        return _popcornWrapper.generatePopcornString( {}, _url, _target, null, callbacks, scripts, collectedEvents );
```
Comment (humphd) [inline src/core/media.js]: This worries me, since it loses anything else that might have been on _popcornOptions.  Why not just explicitly set rAF on that options object to false?

Comment (dseif) [inline src/core/media.js]: I don't think we set anything else on the options object but you're right in that I probably shouldn't ignore the entire object. I'll update now.

File: src/core/media.js [hunk 2]
```diff
@@ -261,6 +261,10 @@
           collectedEvents = collectedEvents.concat( _tracks[ i ].trackEvents );
         }
 
+        /* TODO: determine if we need to turn on frameAnimation or not before calling generatePopcornString
```
Comment (humphd) [inline src/core/media.js:264]: File a bug on this TODO so we don't lose it.  Put the bug # in the comment.

File: src/core/popcorn-wrapper.js [hunk 3]
```diff
@@ -283,7 +283,7 @@ define( [ "core/logger", "core/eventmanager" ], function( Logger, EventManager )
       // special case for basePlayer, since it doesn't require as much of a harness
       if( _mediaType === "baseplayer" ) {
         popcornString +=  "Popcorn.player( 'baseplayer' );\n" +
-                          "var popcorn = Popcorn.baseplayer( '#" + target + "', " + popcornOptions + " );\n";
+                          "var popcorn = Popcorn.baseplayer( '#" + target + "' " + popcornOptions + " );\n";
```
Comment (cadecairos) [inline src/core/popcorn-wrapper.js:286]: Why did you remove that comma?

Comment (dseif) [inline src/core/popcorn-wrapper.js:286]: because it shouldn't be there. Popcorn options inserts it one for us, so having two would be wrong. I just saw it and fixed it, as it's going to have to get fixed sooner or later for the baseplayer to work.
